app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/network/MarvelApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.network
```

Comment:
This is similar to what i meant, to put classes into domain packages like `order` and then if given domain grows create packages for layers like `order.model` `order.presenter` `order.view`. I am not against creating technical packages like `mvp` with boilerplate base classes that are not domain specific.